### PR TITLE
Include "GitHub Usage"

### PR DESCRIPTION
**Pull request by loic-sharma**
Addresses https://github.com/nuget/nugetgallery/issues/7361

/cc @mogah I don't know why, but I can't list you as a reviewer...

### CODE REVIEW

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[5d93394](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1587/commits/5d9339475241acdf998a94050a7583178887f296)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/consume-packages/Finding-and-Choosing-Packages.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/loic-sharma-patch-1/docs/consume-packages/Finding-and-Choosing-Packages.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/consume-packages/finding-and-choosing-packages?branch=pr-en-us-1587) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C8%5C1%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201908011831365028-1587%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=88uM9DFImQ1lfW6JFaLhuHBhnkHBNXblDojY2BN%2BtVc%3D&st=2019-08-01T18%3A27%3A44Z&se=2019-09-01T18%3A32%3A44Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[4b141bc](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1587/commits/4b141bc1bd7e52fee0f6b88f9a8dc6165122eaa5)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/consume-packages/Finding-and-Choosing-Packages.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/loic-sharma-patch-1/docs/consume-packages/Finding-and-Choosing-Packages.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/consume-packages/finding-and-choosing-packages?branch=pr-en-us-1587) |
[docs/consume-packages/media/GitHub-Usage.png](https://github.com/NuGet/docs.microsoft.com-nuget/blob/loic-sharma-patch-1/docs/consume-packages/media/GitHub-Usage.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/consume-packages/media/github-usage.png?branch=pr-en-us-1587) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C8%5C1%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201908011840547365-1587%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=s68%2BR0rRujJg9mPZF9dh%2BIxKZOhE1aFGb%2Fy4TWHFe58%3D&st=2019-08-01T18%3A36%3A59Z&se=2019-09-01T18%3A41%3A59Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[375da6b](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1587/commits/375da6b64f487005cc8a7cf74995d0f014310db3)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/consume-packages/Finding-and-Choosing-Packages.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/loic-sharma-patch-1/docs/consume-packages/Finding-and-Choosing-Packages.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/consume-packages/finding-and-choosing-packages?branch=pr-en-us-1587) |
[docs/consume-packages/media/GitHub-Usage.png](https://github.com/NuGet/docs.microsoft.com-nuget/blob/loic-sharma-patch-1/docs/consume-packages/media/GitHub-Usage.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/consume-packages/media/github-usage.png?branch=pr-en-us-1587) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C8%5C1%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201908012001453360-1587%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=6eYaht0AeyOyMu7J3gw0jcN42Wv2qOTqWzSj%2B6VQJkc%3D&st=2019-08-01T19%3A57%3A49Z&se=2019-09-01T20%3A02%3A49Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/consume-packages/Finding-and-Choosing-Packages.md [hunk 1]
```diff
@@ -77,6 +77,10 @@ At the same time, using a NuGet package means taking a dependency on it, so you
 
     ![Download statistics on a package's listing page](media/Finding-03-Downloads.png)
 
+- *GitHub Usage*: on the package page, the **GitHub Usage** sections lists the top GitHub repositories that depend on this package. A package that is depended on by many popular GitHub repositories is typically a better choice.
```
Comment (Mikejo5000) [inline docs/consume-packages/Finding-and-Choosing-Packages.md]:
```suggestion
- *GitHub Usage*: on the package page, the **GitHub Usage** section lists the top GitHub repositories that depend on this package. A package that many popular GitHub repositories depend on is typically a better choice.
```

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[98a4e74](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1587/commits/98a4e740d1aebdea90c2b20f298352fec4caea86)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/consume-packages/Finding-and-Choosing-Packages.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/loic-sharma-patch-1/docs/consume-packages/Finding-and-Choosing-Packages.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/consume-packages/finding-and-choosing-packages?branch=pr-en-us-1587) |
[docs/consume-packages/media/GitHub-Usage.png](https://github.com/NuGet/docs.microsoft.com-nuget/blob/loic-sharma-patch-1/docs/consume-packages/media/GitHub-Usage.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/consume-packages/media/github-usage.png?branch=pr-en-us-1587) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C8%5C2%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201908021448417269-1587%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=WfCr9app4Ra5uCYXZAr0oSaBlMFUDlyLnf0oSsFiDjU%3D&st=2019-08-02T14%3A44%3A41Z&se=2019-09-02T14%3A49%3A41Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**